content/en/apps/reference/forms/contact.md

```diff
@@ -7,39 +7,76 @@ description: >
+{{% alert title="Note" %}}
+Prior to version 3.7, CHT Core supported 4 contact types - 3 place types (`clinic`, `health_center`, `district_hospital`) and one person type (`person`). From version 3.7, an unlimited number of place and person types are supported. To learn more about setting up configurable hierarchy, review the [Hierarchy]({{< ref "apps/reference/app-settings/hierarchy" >}}) page.
 
+When "contact_type id" is used below, it is referring to the id of the contact type configured in the `app_settings.json` (if using configurable hierarchy) or the relevant `clinic`, `health_center`, `district_hospital`, or `person` value for the contact in context.
+{{% /alert %}}
 
-    - /forms/contact/person-create.xlsx
-    - /forms/contact/person-edit.xlsx
-    - /forms/contact/PLACE_TYPE-create.xlsx
-    - /forms/contact/PLACE_TYPE-edit.xlsx
-    - /forms/contact/places.json
+These forms are stored in the `forms/contact` sub-folder of the project config directory. The naming convention used should be `<contact_type_id-{create|edit}>.xlsx`.
 
-Here is an example of a `places.json` files: 
+## Form details
 
-    {
-      "clinic": "Village Clinic",
-      "health_center": "Health Center",
-      "district_hospital": "District Hospital"
-    }
+### Survey sheet
+
+To collect information about the contact, use a top-level group with the contact_type id as the name (e.g. `person` when adding or editing a person contact). Information in this group will be saved to the contact's document in the database.
+
+| type        | name   | label::en    |
+|-------------|--------|--------------|
+| begin group | person | NO_LABEL     |
+| hidden      | parent | Parent Id    |
+| hidden      | type   | Contact Type |
+| string      | name   | Full Name    |
+| ...         |        |              |
+| end group   |        |              |
+
+The `parent`, `type`, and `name` fields are mandatory on forms that are adding contacts. `parent` will be automatically populated with the id of the parent contact. `type` will be automatically set to the contact_type id when saving the new contact. 
+
+{{% alert title="Edit forms" %}}
+For edit forms, the name of the top-level group should still match the contact_type id of the contact, but only the relevant fields for editing need to be specified in the form.
+{{% /alert %}}
+
+### Settings sheet
+
+The `form_id` should follow the pattern `contact:CONTACT_TYPE_ID:ACTION` where CONTACT_TYPE_ID is the contact_type id for the contact and ACTION is `create` or `edit`. (e.g. `contact:clinic:create`)
+
+### Properties
+
+Starting in cht-core release 3.10, we can now configure property files in contact forms to show or hide them based on an expression or permission as specified in the [app form schema]({{< ref "apps/reference/forms/app#formsappform_namepropertiesjson" >}}).
+
+## Generic contact forms
```

Comment:
One additional thought I had was that in the future if we wanted to expand/improve the documentation in this section (and the one below it) these would seem to be good candidates to move into their own "Quick Guides" pages under `guides/forms`.

I have kept them in here for now to try and avoid too much creep in this PR, but I am happy to move one or both if folks think this reference page is already too big as it is....